powdr_cli/Cargo.toml

```diff
@@ -3,6 +3,9 @@ name = "powdr_cli"
 version = "0.1.0"
 edition = "2021"
 
+[features]
+halo2 = ["dep:halo2"]
```

Comment:
Why is it `dep:halo2` instead of just `halo2`?

Reply:
Right, in this case it would be equivalent to just activate the implicit `halo2` feature.